app/assets/stylesheets/pictures.scss

```diff
@@ -0,0 +1,112 @@
+// Place all the styles related to the pictures controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+a:link    {color:#000;} 
+a:visited {color:#000;}  
+a:hover   {color:#000;}  
+a:active  {color:#000;}
+
+body {
+  background-color: #F0F0F0;
+  font-family: sans-serif;   
+}
+
+.login {
+  position: relative;
```

Comment:
Properties should be ordered background-color, border-color, border-style, border-width, margin, margin-bottom, margin-top, max-width, padding, position